packages/variants/src/variants.ts

```diff
@@ -14,6 +15,27 @@ function makeThemeFromPartials(
   });
 }
 
+/**
+ * Sets the variant theme based on which variant is currently applied.
+ *
+ * @export
+ * @param {IPartialTheme} theme the theme to build a variant for
+ * @param {VariantThemeType} variant the variant type designation
+ * @returns {ITheme} the variant theme
+ */
+export function getVariant(theme: ITheme, variant: VariantThemeType): ITheme {
+  switch (variant) {
+    case VariantThemeType.Neutral:
+      return getNeutralVariant(theme);
+    case VariantThemeType.Soft:
+      return getSoftVariant(theme);
+    case VariantThemeType.Strong:
+      return getStrongVariant(theme);
+    default:
+      return theme;
+  }
+}
+
 /**
  * A variant where the background soft shade of the neutral color. Most other colors remain unchanged.
```

Comment:
Compared to line 92 this sentence is missing a `is`.

Reply:
I went ahead and updated the sentences so they match.